website/docs/guides/migration/versions/04-upgrading-to-dbt-utils-v1.0.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Migrating to dbt utils 1.0"
```

Comment:
Adding the below to be consistent with the sidebar, but wanted to double check with you if 'upgrading' is the correct term? or is it 'migrating'? Also, should we use v1.0 so it reads like the rest of the migration guides?

```suggestion
title: "Upgrating to dbt utils 1.0"
```